assets/javascripts/discourse/components/ai-features-list.gjs

```diff
@@ -61,11 +66,11 @@ class ExpandableList extends Component {
 
 export default class AiFeaturesList extends Component {
   get sortedModules() {
-    return this.args.modules.sort((a, b) => {
-      const nameA = i18n(`discourse_ai.features.${a.module_name}.name`);
-      const nameB = i18n(`discourse_ai.features.${b.module_name}.name`);
-      return nameA.localeCompare(nameB);
-    });
+    if (!this.args.modules || !this.args.modules.length) {
+      return [];
+    }
+
+    return this.args.modules.sortBy("module_name");
```

Comment:
one thing to keep mindful of is that sortBy is going to go eventually. @CvX what is the clean and quick way of doing this in a future safe way?